### PR TITLE
test: use codecov/codecov-action

### PR DESCRIPTION
**Pull request by Layouwen**
https://github.com/ant-design/ant-design/issues/50387
https://github.com/ant-design/ant-design/issues/50461

### CODE REVIEW

**Comment by Layouwen** [comment]
image

remove by backspace key 的测试用例似乎过不去 @zombieJ 有空可以看看, 或者不急的话我把其他 ci 检查后回头看看~

File: package.json [hunk 1]
```diff
@@ -75,7 +75,8 @@
     "rc-virtual-list": "^3.0.0",
     "react": "^16.8.0",
     "react-dom": "^16.8.0",
-    "typescript": "^5.0.0"
+    "typescript": "^5.0.0",
+    "cheerio": "1.0.0-rc.12"
```
Comment (afc163) [inline package.json:79]: https://github.com/cheeriojs/cheerio/issues/4014

File: tests/Select.multiple.spec.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ describe('TreeSelect.multiple', () => {
 
   it('remove by backspace key', () => {
     const wrapper = mount(createSelect({ defaultValue: ['0', '1'] }));
-    wrapper.find('input').first().simulate('keyDown', { which: KeyCode.BACKSPACE });
+    wrapper.find('input').first().simulate('keyDown', { which: KeyCode.BACKSPACE, key: 'Backspace' });
```
Comment (afc163) [inline tests/Select.multiple.spec.js:35]: https://github.com/react-component/select/blob/3927d01c795d7ed5caef083706ed5ce52fec0fd6/src/BaseSelect/index.tsx#L507

File: tests/Select.checkable.spec.tsx [hunk 3]
```diff
@@ -489,7 +489,7 @@ describe('TreeSelect.checkable', () => {
   describe('labelInValue', () => {
     it('basic', () => {
       const wrapper = mount(
-        <TreeSelect checkable labelInValue value={[{ value: '0-0' }]}>
+        <TreeSelect treeCheckable labelInValue value={[{ value: '0-0' }]}>
```
Comment (afc163) [inline tests/Select.checkable.spec.tsx:492]:
```suggestion
        <TreeSelect labelInValue value={[{ value: '0-0' }]}>
```

File: tests/Select.checkable.spec.tsx [hunk 4]
```diff
@@ -489,7 +489,7 @@ describe('TreeSelect.checkable', () => {
   describe('labelInValue', () => {
     it('basic', () => {
       const wrapper = mount(
-        <TreeSelect checkable labelInValue value={[{ value: '0-0' }]}>
+        <TreeSelect labelInValue value={[{ value: '0-0' }]}>
```
Comment (afc163) [inline tests/Select.checkable.spec.tsx:492]:
```suggestion
        <TreeSelect treeCheckable showCheckedStrategy="SHOW_PARENT" labelInValue value={[{ value: '0-0' }]}>
```

**Pull request merged**